pkgs/applications/audio/mbrola/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
     sha256 = "1w86gv6zs2cbr0731n49z8v6xxw0g8b0hzyv2iqb9mqcfh38l8zy";
   };
 
+  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
```

Comment:
Please add a comment that this is needed for it to cross-cc.

I can easily see myself removing this in a version bump because it looks superfluous.

Reply:
this.